doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -18,6 +18,17 @@ EVP_PKEY_param_fromdata_settable, EVP_PKEY_key_fromdata_settable
 
 =head1 DESCRIPTION
 
+The function described here are used to create new keys from user
+provided key data, such as I<n>, I<e> and I<d> for a minimal RSA
+keypair.
+
+These functions use an B<EVP_PKEY_CTX> context, which should primarly
+be created with L<EVP_PKEY_CTX_new_from_name(3)> or
+L<EVP_PKEY_CTX_new_id(3)>.
+
+The exact key data that the user can pass depends on the key type.
+These are passed as an L<OSSL_PARAM(3)> array.
+
 EVP_PKEY_param_fromdata_init() initializes a public key algorithm context
 for creating key parameters from user data.
 
```

Comment:
I realised the reference below to `the "Built-in RSA Import/Export Types" section on the L<provider-keymgmt(7)> page` isn't quite right - because we specifically reference the RSA import/export types, but the text is referring to all key types. The wording probably needs to be adjusted to say something about that section and the following sections...or something along those lines.

Reply:
We should have a *different* page for our providers.  `provider-*.pod` is not meant to hold the implementation details for our providers, they are meant as a description for those who want to write their own providers, which is different.

Reply:
These parameters sit on the interface between providers and applications. They are sent by applications and received by providers. It would seem very strange to have the same things documented twice. I don't mind too much *where* they are documented if you want to move them. But right at the moment they sit in `provider-keymgmt(7)` - so the reference should be right.

Reply:
I didn't say they should be duplicated...  they should indeed be moved.

Reply:
Done in #11270, that should resolve this particular concern, no?

Reply:
@mattcaswell is your concern resolved?

Reply:
Yes